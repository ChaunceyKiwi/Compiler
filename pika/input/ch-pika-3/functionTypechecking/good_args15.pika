func simple <[int] x, float y> -> bool {
  print x; y; _n_.
  return _true_.
}

exec {
  const turf := simple([7, 21], 44.4).
}
