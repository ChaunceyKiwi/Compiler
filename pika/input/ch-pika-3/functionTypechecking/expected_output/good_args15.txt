[7, 21] 44.4 
